rust/core-lib/src/file.rs

```diff
@@ -259,8 +259,8 @@ where
 
                 new_tail_details = TailDetails {
                     current_position_in_tail: end_position,
-                    is_tail_enabled: true,
-                    is_at_bottom_of_file: true,
+                    is_tail_enabled: v.tail_details.is_tail_enabled,
+                    is_at_bottom_of_file: v.tail_details.is_at_bottom_of_file,
```

Comment:
Would it be better to add a new fn in FileManager to update tail position instead of creating a new one each time?

Reply:
Can you explain more what you mean?

Reply:
The only difference between `new_tail_details` and old `new_tail_details` is the `current_position_in_tail`. Instead of creating a new `TailDetails` one each time, I was thinking maybe I can just update `current_position_in_tail` of existing one.